specs/order_spec.rb

```diff
@@ -78,32 +78,84 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
+
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
-      # Useful checks might include:
-      #   - Order.all returns an array
-      #   - Everything in the array is an Order
-      #   - The number of orders is correct
-      #   - The ID and products of the first and last
-      #       orders match what's in the CSV file
-      # Feel free to split this into multiple tests if needed
+      orders = Grocery::Order.all
+      orders.must_be_kind_of Array
+    end
+
+    it "checks if orders match what is the CSV file" do
+      orders = Grocery::Order.all
+      csv = CSV.read("./support/orders.csv")
+      csv_product = csv.to_a[18][1].gsub(/:|;/, " ")
+      order_product = orders[18].products.flatten.join(" ")
```

Comment:
This is a very concise way to make this check!

As we talked about offline, this bit of logic is pretty complex. It would probably be worthwhile to leave some comments here so that you (or your coworkers) have an idea of what format the data is in, and what your transformations are doing.